### DISCUSSION TITLE
Zotero now provides 64 bit installers.

### DISCUSSION THREAD

**Discussion by hkdemiralp** [chocolatey-community/chocolatey-packages#2522] [General]
Zotero package in chocolatey is 32 bit. But Zotero has started providing 64 bit installers. The upstream encourages to install 64 bit version of Zotero as seen in the image below. I tested 64bit version (v7.0.1) in my local choco overlay, and it just works. Switching to 64bit is seamless. No need to migrate the zotero database.
![image](https://github.com/user-attachments/assets/1779f23e-20e2-4e28-bb1a-f51af084b1fe)

**Comment by teknowledgist** [comment]
A link to the 3rd attempt at a PR to fix this:  https://github.com/chocolatey-community/chocolatey-packages/pull/2636

It is currently stagnant for reasons I don't understand.

**Reply by pauby** [reply]
> It is currently stagnant for reasons I don't understand.

As we've all said a few times now, people work in here in their spare time. This is a community driven effort. 

Nobody has had time to review the PR. There is nothing more to it than that.

I've reviewed the PR and left a few comments asking why some changes were made.

**Reply by teknowledgist** [reply]
That's fair.  I thought maybe the failing check to "add comment to PR" was the cause and something I needed to address before it would be reviewed.

**Comment by rgoltz** [comment]
The PR https://github.com/chocolatey-community/chocolatey-packages/pull/2636 is merged :-) - So, we can close this case here?

**Comment by teknowledgist** [comment]
* The PR is "closed" because it is merged. 
* [The issue](https://github.com/chocolatey-community/chocolatey-packages/issues/2531) is closed. 
* Can discussions (like this) be "closed"? I think we just stop discussing. 😄

**Reply by corbob** [reply]
It does look like we can close them, so I've closed it 😄
